test/wsdl-parse-test.js

```diff
@@ -14,4 +15,13 @@ describe(__filename, function () {
       done();
     });
   });
+
+  it('should only parse a WSDL once', function(done) {
```

Comment:
`createClient` is only using `open_wsdl`.  Seems we ultimately care that `require('soap').createClient` will not fail on a recursive wsdl.